### PR TITLE
Add Parca with Parca Agent for continuous profiling

### PR DESCRIPTION
**Pull request by metalmatze**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

I successfully got Parca up and running with CPU samples ingested by simply running `kubectl apply -f ./parca/app.yaml`. Should be good to go! :rocket:

### CODE REVIEW

File: parca/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: parca
+title: "Parca"
+version: v0.4.1
+maintainer: "matthias.loibl@polarsignals.com"
+description: Continuous profiling for analysis of CPU, memory usage over time, and down to the line number. Saving infrastructure cost, improving performance, and increasing reliability.
```
Comment (saiyam1814) [inline parca/manifest.yaml:6]: Max 160 chars can be put in description

**Comment by saiyam1814** [comment]
Also parca/logo.png is wider and taller than 512px

**Comment by saiyam1814** [comment]
Thank you for the PR @metalmatze just wanted to let you know that only app.yaml gets deployed to the cluster and not the other yaml files. 
Also do you want to put anything from the README to post_install ? because only post install is seen by the user when they deploy the app.

**Comment by metalmatze** [comment]
Yep, the `parca.yaml` and `parca-agent.yaml` are merged into `app.yaml`. It's just that we generated the previous two as part of our Parca pipeline, and then combine them in here.
I don't think that we need anything from the README.md in the post_install.
Thanks for the review!

**Comment by saiyam1814** [comment]
LGTM

**Comment by saiyam1814** [comment]
It won't be visible immediately , I will have to push to production and let you know here once the app is available

**Comment by saiyam1814** [comment]
Its live  in production now !! thank you again for the PR @metalmatze

**Pull request merged**